boot-speed/clouds/measure-cloud.py

```diff
@@ -15,13 +15,66 @@
 import os
 import paramiko
 import pycloudlib
+import shutil
 import sys
 
 from pathlib import Path
 
 
 known_clouds = ['ec2', 'gce']
 
+class EC2Instspec():
+    def __init__(self, release):
+        self.release = release
+
+    # Defaults
+    region = "us-east-1"
+    inst_type = "t2.micro"
+    subnetid = ""
+    sgid = []
+
+    def measure(self, datadir, instances=1, reboots=1):
+        """
+        Measure Amazon AWS EC2.
+        Returns the measurement metadata as a dictionary
+        """
+        print('Perforforming measurement on Amazon EC2')
+
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
Is this so that we see the logs from the libraries we're using? It doesn't look like we're logging anywhere.

(Doing this in `measure` also seems quite far down the stack; should we be doing it in `main()`?)

Reply:
I'm mostly interested in logging what pycloudlib is doing, but I agree it makes more sense to have it in `main()`, so I moved it there. It logs to stderr (and thus to the Jenkins console log).